### PR TITLE
Js task 5

### PR DESCRIPTION
**Pull request by dudzon**
https://mogoscroll.netlify.com/

### CODE REVIEW

File: css_tasks/task11/js/scroll.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import {elements} from './elements.js';
+export function scroll() {
+  elements.then((data) => {
+    const {navbar, header, links} = data;
+
+    // smooth scrolling
+    const handleScroll = (item, e) => {
```
Comment (tkachukivan) [inline css_tasks/task11/js/scroll.js]: handleScroll but all on click looks like something went wrong

why not just pass event and then get target from event?

File: css_tasks/task11/js/scroll.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      e.preventDefault();
+      setActiveLink(item);
```
Comment (tkachukivan) [inline css_tasks/task11/js/scroll.js]: looks strange that this before check for tag

File: css_tasks/task11/js/scroll.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    const setHeaderBackground = () => {
+      window.scrollY >= 100
+        ? (header.style.background = 'grey')
+        : (header.style.background = 'none');
+    };
+    //  Show currently clicked link
+
+    const setActiveLink = (item) => {
+      links.forEach((link) => link.removeAttribute('data-active'));
```
Comment (tkachukivan) [inline css_tasks/task11/js/scroll.js]: same as mention before, that you can save active in some variable

**Review by tkachukivan — COMMENTED** [review]
there is no sync when scrolling with header highlights

I would suggest for you to try implement some functionality using classes

**Review by tsvetkovv — COMMENTED** [review]
nothing more than Ivan said

**Pull request merged**